src/app/views/authentication/Authentication.tsx

```diff
@@ -0,0 +1,142 @@
+import hello from 'hellojs';
+import { PrimaryButton } from 'office-ui-fabric-react';
+import React, { Component } from 'react';
+
+import { connect } from 'react-redux';
+import { bindActionCreators, Dispatch } from 'redux';
+
+import { IAuthenticationProps, IAuthenticationState } from '../../../types/authentication';
+import * as authActionCreators from '../../services/actions/auth-action-creators';
+import * as queryActionCreators from '../../services/actions/query-action-creators';
+import { ADMIN_AUTH_URL, AUTH_URL, DEFAULT_USER_SCOPES , TOKEN_URL, USER_INFO_URL } from '../../services/constants';
+import './authentication.scss';
+
+export class Authentication extends Component<IAuthenticationProps,  IAuthenticationState> {
+  public state = {
+    authenticated: {
+      status: false,
+      user: {
+        displayName: '',
+        emailAddress: '',
+      },
+      token: '',
+    },
+  };
+
+  public componentDidMount = () => {
+    const authenticated = localStorage.getItem('authenticated');
```

Comment:
You should get this information from the store instead of the local storage.

Reply:
the store looses data when the page reloads. Thus on successful authentication, the details are saved in the store and the status also saved in the local storage. The componentDidMount checks whether the user is authenticated and passes the localstorage data to the state so that the user data is still known

Reply:
You should hydrate the store with this information whenever the application loads.